bot/resources/tags/intents.md

```diff
@@ -1,19 +1,17 @@
 **Using intents in discord.py**
 
-Intents are a feature of Discord that tells the gateway exactly which events to send your bot. By default discord.py has all intents enabled except for `Members`, `Message Content`, and `Presences`. These are needed for features such as `on_member` events, to get access to message content, and to get members' statuses.
+Intents are a feature of Discord that tells the gateway exactly which events to send your bot. Various features of discord.py rely on having particular intents enabled. Since discord.py v2.0.0, this has become **mandatory** for developers to define in their code.
 
-To enable one of these intents, you need to first go to the [Discord developer portal](https://discord.com/developers/applications), then to the bot page of your bot's application. Scroll down to the `Privileged Gateway Intents` section, then enable the intents that you need.
-
-Next, in your bot you need to set the intents you want to connect with in the bot's constructor using the `intents` keyword argument, like this:
+There are *standard* intents and *privileged* intents. The current privileged intents are `Presences`, `Server Members`, and `Message Content`. To use one of the privileged intents, you have to first enable them in the [Discord Developer Portal](https://discord.com/developers/applications). Go to the `Bot` page of your application, scroll down to the `Privileged Gateway Intents` section, and enable the privileged intents that you need.
```

Comment:
We don't have access to the standard intents in the developer portal. What do you think about mentioning that ? We'd want to have people ask the question "Why don't I see these standard intents, am I doing something wrong ?"

We could say the the ones we can define in the portal are the privileged ones.
And maybe, if we want, we could add the link to a detailed technical list of these intents.

Reply:
Yeah, pointing to the [discord.Intents](https://discordpy.readthedocs.io/en/stable/api.html#intents) reference is a good idea. I'll link to it in the first paragraph.

I would think that how it's written right now would sufficiently imply that standard intents aren't part of the dev portal, but I can add a short sentence to clarify that.